.github/DISCUSSION_TEMPLATE/feature-requests.yml

```diff
@@ -0,0 +1,8 @@
+body:
+  - type: textarea
+    id: discussion-field
+    attributes:
+      label: Tell us more.
+      description: Explain your feature request in this field.
```

Comment:
- https://github.com/renovatebot/renovate/pull/19894#issuecomment-1397373146

At the time I created the first Discussion forms, the `description:` keys could not be used.

GitHub Docs now use the `description:` key in their Discussion form examples, so I hope it will work for us in production later. 😄 

[^1]: [GitHub Docs, Syntax for discussion category forms](https://docs.github.com/en/discussions/managing-discussions-for-your-community/syntax-for-discussion-category-forms)